Masterbots/UI fixes

AnoukRImola opened the pull request:
In this PR, the following changes were made:

Desktop:

- The class was modified so that the question window remains static regardless of the number of questions:

![image](https://github.com/bitcashorg/masterbots/assets/77553677/23dcb672-ac33-42fc-93cc-5f501e37c333)

Mobile:

-  The sticky was fixed.
-  A scroll was added for when the questions are too long.
-  The right margin was removed as instructed.
-  The Sidebar Icon was modified to appear only when navigating the 'chat' route.
![image](https://github.com/bitcashorg/masterbots/assets/77553677/49cfe4c0-b2b6-44eb-b459-21173ad87ab5)
![image](https://github.com/bitcashorg/masterbots/assets/77553677/f24e938a-788d-427c-8cd0-17f863cac4b5)


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Style Updates**
  - Standardized CSS class names for consistency across multiple components, improving maintainability and readability.
  - Updated width, height, and size class names to new conventions (`size-4`, `size-6`, `size-7`, `size-9`) for several elements.
  - Adjusted maximum width classes to utilize screen size breakpoints (e.g., `max-w-screen-lg`).

- **Refactor**
  - Wrapped `SidebarToggle` in a `React.Suspense` component for improved code splitting and loading states.
  - Introduced `SidebarToggleWrap` to conditionally render based on the current pathname.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

AndlerRL commented on apps/masterbots.ai/components/browse-chat-message-list.tsx:
```diff
@@ -56,7 +56,7 @@ export function BrowseChatMessageList({
           <></>
```
Please clean this line.

AndlerRL commented on apps/masterbots.ai/components/thread-popup.tsx:
```diff
@@ -87,7 +87,7 @@ export function ThreadPopup({ className }: { className?: string }) {
             )}
           </div>
           <button
-            className="absolute right-[32px] top-[50%] translate-y-[-50%] "
+            className="absolute right-[32px] top-1/2 translate-y-1/2 "
             onClick={onClose}
           >
             <IconClose />
```
Good suggestion. @AnoukRImola , can you apply the update? I tried with coderabbitai but it throws errors.

AndlerRL reviewed (approved):
LGTM! I notice only 1 tweak and a clean up. After that, it will be ready for a merge!

AndlerRL commented:
@AnoukRImola -- please handle the conflict with the changes 🙏

AnoukRImola commented:
@AndlerRL ,I cleaned up, updated more Tailwind classes, fixed the conflict with the bun.lockb file, and updated the input texts to align to the left.

![image](https://github.com/bitcashorg/masterbots/assets/77553677/1fc20e17-b26b-4e55-85ce-364c33a139b4)

AndlerRL commented:
@AnoukRImola -- Thanks for taking care of the feedbacks! I see though 2 left from coderabbitai. Also, I noticed a build error on the last commit -> https://vercel.com/bitcash/masterbots/2wHnPgDMxH1pFtbi9rXL9b5Qdmrn?filter=errors can you check that out?

AndlerRL commented:
![](https://github.trello.services/images/mini-trello-icon.png) [[FE] UI/UX Tweaks Hotfixes - Mobile Threads + Dialog](https://trello.com/c/ZOU7im9t/177-fe-ui-ux-tweaks-hotfixes-mobile-threads-dialog)
